main/main.go

```diff
@@ -234,27 +234,67 @@ func main() {
 			)
 			panic(err)
 		}
-		subnetInfo := s
+		sourceBlockchain := s
```

Comment:
Do we need this local var? I think we can just use `s` directly where needed, and maybe rename `s` to `sourceBlockchain` for clarity.

Reply:
Ah, gets me every time...makes sense thanks